java/pom.xml

```diff
@@ -29,6 +29,39 @@
     <tag>HEAD</tag>
   </scm>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>biz.aQute.bnd</groupId>
+        <artifactId>bnd-maven-plugin</artifactId>
+        <version>6.4.0</version>
```

Comment:
@laeubi any reason not to use v7.1?

Reply:
> @laeubi any reason not to use v7.1?

This requires the build to use Java 17, I think that still work to build for java 8 then but don't wanted to change to much at once.

Reply:
Okay, please upgrade and you can drop 11 from the matrix.